client/src/components/AccountManager/index.tsx

```diff
@@ -1,20 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styles from './styles.module.scss';
 import { Button, Icon } from 'semantic-ui-react';
 import { Link } from 'react-router-dom';
+import { removeToken } from 'helpers/setToken.helper';
+import { logOutUserTrigger } from 'containers/LoginPage/logic/actions';
 import { requestDeleteUser } from 'containers/ProfilePage/logiс/actions';
 import { useDispatch } from 'react-redux';
 import LanguageSelect from 'components/LanguageSelect';
 import { useTranslation } from 'react-i18next';
+import DeleteUserModal from 'components/DeleteUserModal';
 
 const AccountManager = () => {
 	const { t } = useTranslation();
 	const dispatch = useDispatch();
-	const DeleteUser = () => {
+	const [isDelete, setIsDelete] = useState(false);
+
+	const onClose = () => {
+		setIsDelete(false);
+	};
+
+	const showDeleteModal = () => {
+		setIsDelete(true);
+	};
```

Comment:
you may change this 2 functions with 1
const toggleModal = () => setIsDelete(!isDelete)